robot/impl/resource_manager.go

```diff
@@ -945,7 +993,17 @@ func (manager *resourceManager) updateResources(
 					resToClose.Name().String(), "module", mod.Name, "error", err)
 			}
 		}
+		if err := manager.removeUniqueResource(mod.Name); err != nil {
+			allErrs = multierr.Combine(allErrs, err)
+		}
+	}
+
+	for _, p := range conf.Removed.Packages {
```

Comment:
Again, I think we can ignore packages and leave uniqueness up to package managers.

Reply:
package manager doesn't handle uniqueness right now, so i feel like adding this here makes it complete. (I did do manual testing and worked correctly).